CONTRIBUTING.md

```diff
@@ -223,6 +223,16 @@ See [`apm-agent-core/README.md`](apm-agent-core/README.md)
 
 See [`apm-agent-plugins/README.md`](apm-agent-plugins/README.md)
 
+### Documenting
+
+Documentation is generated text files stored in `docs` folder using [AsciiDoc](http://asciidoc.org/) format.
```

Comment:
Only the configuration documentation (`configuration.asciidoc`) is generated, the rest is manually written